pkg/minikube/constants/constants.go

```diff
@@ -39,4 +39,14 @@ func MakeMiniPath(fileName string) string {
 	return filepath.Join(Minipath, fileName)
 }
 
+var LogFlags = [...]string{
+	"logtostderr",
+	"alsologtostderr",
+	"stderrthreshold",
+	"logdir",
+	"logbacktraceat",
+	"v",
+	"vmodule",
```

Comment:
These are optional flags for configuring glog and are intended only for internal use